.github/actions/spelling/patterns.txt

```diff
@@ -1,21 +1,9 @@
-[Ll][Oo][Mm][Bb][Ii][Qq]
```

Comment:
The [documentation examples](https://github.com/check-spelling/check-spelling/wiki/Configuration-Examples%3A-patterns) actually use comments, which I think makes a lot of sense in combination with Regexes.